test/extended/testdata/jenkins-plugin/build-job.xml

```diff
@@ -27,6 +27,22 @@
       <authToken></authToken>
       <followLog>true</followLog>
       <checkForTriggeredDeployments>true</checkForTriggeredDeployments>
+
```

Comment:
My intention was indeed to minimize time, but I've split it into two tests per your request.